postgresql/manifests/upstream/delete-secret.yaml

```diff
@@ -0,0 +1,6 @@
+$patch: delete
```

Comment:
I see this secret is used here: https://github.com/utilitywarehouse/shared-kustomize-bases/pull/12/files#diff-f93ae1c7b13f03b05eab33794ab4836900c63c7374a48b1099e085b60edd1a28R176-R180 . 
I guess that if we don't want it generated, we need to provide a secret name as the value `auth.existingSecret` when generating the template

Reply:
Done!